Sources/NIOWebSocket/WebSocketFrame.swift

```diff
@@ -71,6 +72,27 @@ extension WebSocketMaskingKey: ExpressibleByArrayLiteral {
     }
 }
 
+extension WebSocketMaskingKey {
+    /// Returns a random masking key, using the given generator as a source for randomness.
+    /// - Parameter generator: The random number generator to use when creating the
+    ///     new random masking key.
+    /// - Returns: A random masking key
+    @inlinable
+    public static func random<T>(
+        using generator: inout T
+    ) -> WebSocketMaskingKey where T: RandomNumberGenerator {
```

Comment:
```suggestion
    public static func random<Generator: RandomNumberGenerator>(
        using generator: inout Generator
    ) -> WebSocketMaskingKey {
```

I think `Generator` is a nicer type parameter name than `T`.